### ISSUE TITLE
How to identify users on Instabug 8.0.x

### ISSUE THREAD

**Issue by marcelofabri** [Instabug/Instabug-iOS#268]
### Steps to Reproduce the Problem

- Update Instabug to 8.0.x
- Try to use `Instabug.setUserAttribute` and get a deprecation warning
- Look at the [migration guide](https://docs.instabug.com/v1.0/docs/ios-migration-guide) and don't find anything about user identification being deprecated
- Look at https://docs.instabug.com/v1.0/docs/ios-identify-user and see that `Instabug.setUserAttribute` is still being used

### Expected Behavior

a)  `Instabug.setUserAttribute` wouldn't be deprecated

or 

b) Another API would be available and mentioned on the migration guide

### Actual Behavior

 `Instabug.setUserAttribute` is deprecated

### Instabug integration code

 `Instabug.setUserAttribute("1", withKey: "foo")`

### SDK Version

8.0.2

### iOS Version

11.4

### Device Type

Doesn't matter

**Comment by zierka** [comment]
I'm having this issue as well. Maybe they deprecated it by mistake. 

A possible way of doing it is to set the `Instabug.willSendReportHandler` closure and configure the `IBGReport` instance that's going to be sent. The `IBGReport` has a `setUserAttribute(_ userAttribute: String, withKey: String)` method.

**Comment by Kmohamed** [comment]
@marcelofabri @zierka Yes it is deprecated by mistake and we are working on a new release now.
We are so sorry for that.

**Comment by zierka** [comment]
@Kmohamed alright, thanks for the fast turnaround!

**Comment by marcelofabri** [comment]
@Kmohamed Thanks! 🎉
